vendor/bat-native-ledger/src/bat/ledger/internal/database/database_migration.cc

```diff
@@ -130,7 +131,8 @@ void DatabaseMigration::Start(uint32_t table_version,
                                           migration::v31,
                                           migration_v32,
                                           migration::v33,
-                                          migration::v34};
+                                          migration::v34,
+                                          migration::v35};
```

Comment:
Would you mind adding a unit test to `vendor/bat-native-ledger/src/bat/ledger/internal/database/database_migration_unittest.cc`?

Reply:
Yes, I'll add one for the v34 migration as well (it seems to be missing).

Reply:
@emerick Already adding one [here](https://github.com/brave/brave-core/compare/sszaloki-24189-fix-transaction-handling-for-all-custodians). You can copy it from there, if you want. 🙂

Reply:
Even better, thanks!

Reply:
All set now!